### PR TITLE
Use a Consistent Postgis Version

### PR DESCRIPTION
**Pull request by charlie-tf**
The postgis version specified for tests and github is inconsistent.
It should be `postgis:13-3.0` to match what is used for staging/prod
deployment in RDS. Also add `.DS_Store` files to `.gitignore`.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ out
 .idea
 openapi.yaml
 .env
+*.DS_Store
```
Comment (sgrimm) [inline .gitignore]: Why the `*` prefix?

**Review by sgrimm — COMMENTED** [review]
Not sure why the CI checks aren't running. Maybe related to the GitHub issues. This seems good (aside from one question) but I'll wait until we have a successful CI run before approving it.

File: .gitignore (hunk 1 above)
Comment (charlie-tf) [reply .gitignore]: to catch when it shows up in subdirectories. I had looked at the repo in Finder, which generated a breadcrumb trail of `.DS_Store` files following my navigation.

**Comment by charlie-tf** [comment]
> Not sure why the CI checks aren't running. Maybe related to the GitHub issues.

Yeah I wondered about that. I did run tests locally, but certainly want to wait for CI to come back online. Thanks!

File: .gitignore (hunk 1 above)
Comment (sgrimm) [reply .gitignore]: Ah, okay, We shouldn't need that -- unqualified filenames should be matched in any directory at or under the level where the `.gitignore` file lives.

`*.DS_Store` would match `Foo.DS_Store` as a filename.

Comment (sgrimm) [reply .gitignore]: Err, to be more helpful: this can be `.DS_Store` rather than `*.DS_Store`.

Comment (charlie-tf) [reply .gitignore]: Oh, I did not know that. I'll change it. Thanks!

**Review by sgrimm — APPROVED** [review]
CI checks passed.

**Pull request merged**